packages/diagrams/lib/src/cupertino_colors.dart

```diff
@@ -0,0 +1,485 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/cupertino.dart';
+
+import 'diagram_step.dart';
+
+const String _basic_colors = 'cupertino_basic_colors';
+const String _active_colors = 'cupertino_active_colors';
+const String _system_colors_1 = 'cupertino_system_colors_1';
+const String _system_colors_2 = 'cupertino_system_colors_2';
+const String _system_colors_3 = 'cupertino_system_colors_3';
+const String _label_colors = 'cupertino_label_colors';
+const String _background_colors = 'cupertino_background_colors';
+
+List<Map<String, Color>> basicColors = <Map<String, Color>>[
+  <String, Color>{'white': CupertinoColors.white},
+  <String, Color>{'black': CupertinoColors.black},
+  <String, Color>{'lightBackgroundGray': CupertinoColors.lightBackgroundGray},
+  <String, Color>{
+    'extraLightBackgroundGray': CupertinoColors.extraLightBackgroundGray
+  },
+  <String, Color>{'darkBackgroundGray': CupertinoColors.darkBackgroundGray},
```

Comment:
move to background, no?

Reply:
These are `Color` types and other background colors are `CupertinoDynamicColor`.
In fact, all "basic colors" colors are `Color` type as a result, I separated them.

Since background colors are `CupertinoDynamicColor`, they have all 4 color variants,  while basics have single value 
![Screenshot 2022-05-10 at 11 08 05](https://user-images.githubusercontent.com/48603081/167580738-36612ebd-818a-47bd-bba6-2f996b2b9e11.png)

Reply:
I also notice basic colors are separated from background colors in the source code

https://github.com/flutter/flutter/blob/6428624c10424e846491f51422ebe6aa16278a6d/packages/flutter/lib/src/cupertino/colors.dart#L63-L76

https://github.com/flutter/flutter/blob/6428624c10424e846491f51422ebe6aa16278a6d/packages/flutter/lib/src/cupertino/colors.dart#L435-L449

Reply:
"  /// Used in iOS 10 for light background fills such as the chat bubble background.
  ///
  /// This is SystemLightGrayColor in the iOS palette.
  static const Color lightBackgroundGray = Color(0xFFE5E5EA);"
 
 and 

"  /// The color for the main background of your interface, equivalent to
  /// [UIColor.systemBackground](https://developer.apple.com/documentation/uikit/uicolor/3173140-systembackground).
  ///
  /// Typically used for designs that have a white primary background in a light environment"

Reply:
I see, yeah. Thanks for digging into it